DESCRIPTION

```diff
@@ -82,6 +83,9 @@ Imports:
     glmnet,
     Rsamtools,
     methods,
-    Rcpp
+    Rcpp,
+    stringr,
+    BSgenome,
```

Comment:
BSgenome is required for getSeq()
stringr is used for str_split() and str_count()

The former is used to get the genome sequence at the edges of the read classes so I can count the number of A's and T's. Do you know another function I can use to avoid importing these packages?